src/octoprint/templates/snippets/settings/webcam/webcamStreamWebrtcIce.jinja2

```diff
@@ -0,0 +1,7 @@
+<div class="control-group" title="{{ _('Endpoint to use for WebRTC ICE')|edq }}">
```

Comment:
I think `ICE server` would be slightly more consistent terminology. It could also be made optional. Taking a stab at updating the copy here: `ICE server (STUN and/or TURN) to use for WebRTC (optional)`

if a STUN server isn't used clients may still be able to negotiate a direct connection but it depends how what layer of network are between them. Even on my LAN I've noticed that STUN makes connections more reliable.


https://datatracker.ietf.org/doc/html/rfc8489
https://developer.mozilla.org/en-US/docs/Web/API/RTCConfiguration/iceServers

Reply:
Rephrased, does that look better?

Reply:
Looks great!